### PR TITLE
feat(api): add support for Topics API

### PR DESCRIPTION
**Pull request by nejch**

### CODE REVIEW

File: .renovaterc.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "regexManagers": [
     {
-      "fileMatch": ["^tests/functional/fixtures/.env$"],
+      "fileMatch": ["^tests\\/functional\\/fixtures\\/.env$"],
```
Comment (JohnVillalovos) [inline .renovaterc.json:10]: nit:  Use r-strings

```
"fileMatch": [r"^tests\/functional\/fixtures\/.env$"],
```

**Review by JohnVillalovos — APPROVED** [review]
I'm okay with it as is, but I think r-strings usage would be good. But not something to stop this from merging.

Thanks!

File: .renovaterc.json (hunk 1 above)
Comment (nejch) [reply .renovaterc.json:10]: Thanks @JohnVillalovos. This is a JSON file though, I don't think that's supported?

Comment (JohnVillalovos) [reply .renovaterc.json:10]: Duh! I was just assuming Python 😟

**Pull request merged**